hilgendorf_moveit_config/config/kinematics.yaml

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
This is as produced by the setup_assistant. I'm happy to make any necessary changes, but I need some reference for it. Is the copy pasting from https://github.com/ros-industrial/universal_robot/blob/hydro-devel/ur5_moveit_config/config/kinematics.yaml reasonable?